srcpkgs/luastatus/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'luastatus'
+pkgname=luastatus
+version=0.6.0
+revision=1
+build_style=cmake
+hostmakedepends="python-docutils pkg-config git"
+makedepends="lua-devel alsa-lib-devel glib-devel libnl3-devel eudev-libudev-devel libX11-devel\
+	yajl-devel libxcb-devel xcb-util-devel xcb-util-wm-devel"
+short_desc="Universal status bar content generator"
+maintainer="Gadzhi Kharkharov <me@kkga.me>"
+license="GPL-3.0-or-later"
```

Comment:
It seems to be LGPL? https://github.com/shdown/luastatus/blob/master/luastatus/luastatus.c

Reply:
Not sure which one should be used, since the github page says GPL-3.0: https://github.com/shdown/luastatus/blob/master/COPYING.txt